extensions/npm/main.ts

```diff
@@ -56,7 +56,16 @@ if (Deno.args.length >= 1
     console.log(`[${green("phylum")}] Installing without build scripts…`);
 
     // Install packages without executing build scripts.
-    let status = await Deno.run({ cmd: ['npm', ...Deno.args, '--ignore-scripts']}).status();
+    let status = PhylumApi.runSandboxed({
+        cmd: 'npm',
+        args: [...Deno.args, '--ignore-scripts'],
+        exceptions: {
+            write: ['~/.npm', './'],
+            run: ['npm'],
+            read: ['~/.npm', './'],
```

Comment:
I'd always cluster read/write together and just move run above/below the two. Especially because they're likely to share some content. Will look much cleaner.

Reply:
Agreed -- this is probably an oversight left over from to testing different stuff.